.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,43 @@
+<!-- Thanks for filing a yew issue! We appreciate it :-) -->
+
+#### Description
+
+**I'm submitting a ...**  (check one with "x")
+
+[ ] question <!-- please describe the problem you're trying to solve and give as much context as possible. -->
```

Comment:
There must be a dash to let markdown render this as checkbox. https://github.com/ctjhoa/yew/blob/2a27296b4514997f8b0bec6101c0474b2150b945/.github/ISSUE_TEMPLATE.md

What about to use tags instead of checkboxes? We tried to use checkboxes in other projects, but it looked everytime like uncompleted issue.

Reply:
Other things looks good 👍

Reply:
I'm not sure to understand what you mean by "tags".
Do you mean github labels ? If yes then I'm not sure that a submitter can add github label.
Or do you mean text tags like `[QUESTION]` ?